packages/cloudvision-connector/src/Wrpc.ts

```diff
@@ -330,7 +331,11 @@ export default class Wrpc {
 
       let msg: CloudVisionMessage;
       try {
-        msg = this.Parser.parse(event.data, this.connectorOptions.batchResults);
+        msg = this.Parser.parse(
+          event.data,
+          this.connectorOptions.batchResults,
+          this.connectorOptions.nanosecondSupport ?? false,
```

Comment:
you set the default value as false in the function, no need to include a fallback value here.